src/cli/download_tracker.rs

```diff
@@ -233,3 +225,42 @@ impl fmt::Display for Size {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    #[test]
+    fn download_tracker_from_seconds_test() {
+        use crate::download_tracker::DownloadTracker;
+        assert_eq!(
+            DownloadTracker::from_seconds(2),
+            (0, 0, 0, 2)
+        );
```

Comment:
The reason the CI test fails is that these are not formatted in the manner rustfmt prefers.  You can run `cargo fmt` to correct things prior to committing your code.

Reply:
Gocha. Much appreciated for your quick response. Will make adjustments now. Thanks!